infra/modules/service/variables.tf

```diff
@@ -170,13 +170,13 @@ variable "vpc_id" {
   description = "Uniquely identifies the VPC."
 }
 
-# Custom Template-diverging variables
-variable "container_read_only" {
-  type        = bool
-  description = "Whether the container root filesystem should be read-only"
-  default     = true
+variable "ephemeral_write_volumes" {
+  type        = set(string)
+  description = "A set of absolute paths in the container to be mounted as writable for the life of the task. These should also be declared with VOLUME statements in the container definition file."
+  default     = []
 }
 
+# Custom Template-diverging variables
```

Comment:
Question unrelated to this PR, but Are these needed for template-application-rails to work? Should we get rid of these template-divergent variables if they aren't needed?

Reply:
> Are these needed for template-application-rails to work

Yes, as noted in 6274c142785a73a622852f9b9ed2df0c2f61102c (though the healthcheck bit/curl support is a hidden requirement not listed in upstream docs/not strictest of requires if we were to change the rails Dockerfile to include `wget`). But once this PR and https://github.com/navapbc/platform-test/pull/176 are merged everywhere, `template-application-rails` won't require infra edits (besides configuring `ephemeral_write_volumes`).